src/main/java/umk/mat/pajda/ProjektZespolowy/entity/ReviewArchive.java

```diff
@@ -0,0 +1,36 @@
+package umk.mat.pajda.ProjektZespolowy.entity;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.PastOrPresent;
+import jakarta.validation.constraints.Size;
+import java.time.LocalDateTime;
+import lombok.*;
+
+@Entity
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class ReviewArchive {
+
+  @Id @GeneratedValue @NotNull private Integer id;
+
+  @Size(min = 0, max = 10)
```

Comment:
default value of min is 0, so its not necessary

Reply:
you're right again